conode/run_conode.sh

```diff
@@ -255,6 +267,14 @@ migrate(){
 			fi
 			echo $VERSION > $PATH_VERSION
 			;;
+		1.2-1)
```

Comment:
I really think this should not be `v1`, but rather `v2-alpha` or so. We're breaking existing code here!